www/_posts/2025-08-29-cordova-fetch-5.0.0.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Fetch 5.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `cordova-fetch@5.0.0` has been released. This is the library used behind-the-scenes of the Cordova tooling for fetching of of Cordova platforms and plugins.
```

Comment:
typo: double `of` in "fetching of of Cordova platform and plugins"

Reply:
Force pushed updates to remove the typo